.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+package.json
```

Comment:
- [ OPTIONAL] I recommend you remove package.json from .gitignore,  It helps to install the packages that a project needs for it to run successfully.